test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -223,4 +222,70 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function() {
       })
       .catch(done.fail);
   });
+
+  it('shows correct text when Can\'t verify? is clicked', function() {
+    spyOn(webauthn, 'isNewApiAvailable').and.callFake(() => true);
+    spyOn(BrowserFeatures, 'isSafari').and.callFake(() => false);
```

Comment:
Please use `jest` objects for `spyOn`:

```suggestion
    jest.spyOn(BrowserFeatures, 'isSafari').mockReturnValue(false);
```